### PR TITLE
Update template for Tauticord

### PR DESCRIPTION
**Pull request by nwithan8**
Update Tauticord.xml for new config (clarify environmental variable names, changes in environmental variables, new volume mappings)

### CODE REVIEW

File: templates/tauticord.xml [hunk 1]
```diff
@@ -14,24 +14,32 @@
...
+    <Config Name="Discord admin IDs" Target="TC_DISCORD_ADMIN_IDS" Default="" Description="Comma-separated list of IDs of Discord users with bot admin privileges" Type="Variable" Display="always" Required="false" Mask="false" />
+    <Config Name="Stream details text channel" Target="TC_DISCORD_CHANNEL_NAME" Default="tauticord" Description="Name of Discord text channel where the bot will post" Type="Variable" Display="always" Required="true" Mask="false">tauticord</Config>
+    <Config Name="Use embeds" Target="TC_USE_EMBEDS" Default="True" Description="Whether to use embeds for Discord messages" Type="Variable" Display="always" Required="false" Mask="false">True</Config>
+    <Config Name="Allow analytics" Target="TC_ALLOW_ANALYTICS" Default="True" Description="Whether to allow anonymous analytics collection" Type="Variable" Display="always" Required="false" Mask="false">True</Config>
+    <Config Name="Timezone" Target="TZ" Default="UTC" Description="Timezone for the server" Type="Variable" Display="always" Required="false" Mask="false">UTC</Config>
```
Comment (Roxedus) [inline templates/tauticord.xml:42]: Redundant

File: templates/tauticord.xml [hunk 2]
```diff
@@ -14,24 +14,32 @@
...
-    <Config Name="TC_DISCORD_CHANNEL_NAME" Target="TC_DISCORD_CHANNEL_NAME" Default="" Description="Discord channel where bot will post" Type="Variable" Display="always" Required="true" Mask="false" />
-    <Config Name="TC_USE_EMBEDS" Target="TC_USE_EMBEDS" Default="True" Description="Whether to use embeds for Discord messages" Type="Variable" Display="always" Required="false" Mask="false" />
-    <Config Name="TC_ALLOW_ANALYTICS" Target="TC_ALLOW_ANALYTICS" Default="True" Description="Whether to allow anonymous analytics collection" Type="Variable" Display="always" Required="false" Mask="false" />
-    <Config Name="TZ" Target="TZ" Default="UTC" Description="Timezone for the server" Type="Variable" Display="always" Required="false" Mask="false" />
+    <Icon>https://raw.githubusercontent.com/nwithan8/tauticord/master/logo.png</Icon>
+    <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/tauticord.xml</TemplateURL>
+    <Config Name="Discord bot token" Target="TC_DISCORD_BOT_TOKEN" Default="" Description="Discord bot token" Type="Variable" Display="always" Required="true" Mask="false" />
```
Comment (Roxedus) [inline templates/tauticord.xml:19]: Probably worth setting this to masked

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: templates/tauticord.xml (hunk 2 above)
Comment (nwithan8) [reply templates/tauticord.xml:19]: Good idea

File: templates/tauticord.xml (hunk 1 above)
Comment (nwithan8) [reply templates/tauticord.xml:42]: What is redundant?

Comment (Roxedus) [reply templates/tauticord.xml:42]: The TZ variable, unraid automatically sets it

**Comment by nwithan8** [comment]
Tauticord configuration has been moved to a different repository.

**Pull request closed without merging**